modules/rules_swift/1.5.0/MODULE.bazel

```diff
@@ -0,0 +1,34 @@
+# Generated by update-module-version.sh. DO NOT EDIT.
+module(
+    name = "rules_swift",
+    compatibility_level = 1,
+    repo_name = "build_bazel_rules_swift",
+    version = "1.4.0",
```

Comment:
Here it should be 1.5.0 already? And we won't need the patch file any more I guess

Reply:
Ah yes, I was trying out of this was causing the hang.

Reply:
Well, we'll still need the patch right? Because as written in the PR description the release archive contains 1.4.0 at the moment.

Reply:
Hmm, I think it already contains 1.5.0, that's why the patch failed.

Reply:
Oh wow yes, I checked the release archive and it contains 1.5.0. I believe we patch it in our release scripts before build the archive, all good then. Removing this patch.